public/_data/repositories/core.yml

```diff
@@ -0,0 +1,39 @@
+section:
+  description: These are the primary repositories for Conjur Open Source and its SDK.
+  categories:
+  - name: Conjur OSS Core
+    description: Conjur OSS Server and Deployment Tools
+    repos:
+      - name: cyberark/conjur
+        url: https://github.com/cyberark/conjur
+        description: Conjur OSS server. Conjur comes built-in with custom authenticators
+          for Kubernetes, OpenShift, AWS IAM, OIDC, and more.
+        thumb: CyberArk_Conjur_logo.svg
+        #image-alt:
+      - name: cyberark/conjur-oss-helm-chart
+        url: https://github.com/cyberark/conjur-oss-helm-chart
+        description: Helm chart for deploying Conjur OSS.
+      - name: cyberark/conjur-aws
+        url: https://github.com/cyberark/conjur-aws
+        description: AWS CloudFormation templates for deploying Conjur OSS.
+  - name: Conjur SDK
+    description: Conjur Command Line Interface (CLI) and Client Libraries
+    repos:
+      - name: cyberark/conjur-cli
+        url: https://github.com/cyberark/conjur-cli
+        description: Conjur Ruby CLI
+      - name: cyberark/conjur-api-dotnet
+        url: https://github.com/cyberark/conjur-api-dotnet
+        description: Conjur .Net Client Library
+      - name: cyberark/conjur-api-go
+        url: https://github.com/cyberark/conjur-api-go
+        description: Conjur Golang Client Library
+      - name: cyberark/conjur-api-java
+        url: https://github.com/cyberark/conjur-api-java
+        description: Conjur Java Client Library
+      - name: cyberark/conjur-api-python3
+        url: https://github.com/cyberark/conjur-api-python3
+        description: Conjur Python Client Library
+      - name: cyberark/conjur-api-ruby
+        url: https://github.com/cyberark/conjur-api-ruby
+        description: Conjur Ruby Client Library
```

Comment:
[NIT] Descriptions for the repos here don't end with '.' while all the other descriptions do (except the repo descriptions in delivery.yml. Repo descriptions in integrations.yml do end with a .).